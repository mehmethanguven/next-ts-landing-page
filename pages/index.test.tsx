import React from 'react'
import { render } from '@testing-library/react'
import Home from './index'

describe('Home test', () => {
  it('should to match snapshot', () => {
    const { asFragment } = render(<Home />)
    expect(asFragment()).toMatchSnapshot()
  })
})
