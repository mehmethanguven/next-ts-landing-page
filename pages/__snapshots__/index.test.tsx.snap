// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home test should to match snapshot 1`] = `
<DocumentFragment>
  <div>
    <section
      id="hero"
    >
      <div
        class="container mx-auto mt-10 flex flex-col-reverse items-center space-y-0 px-6 md:flex-row md:space-y-0"
      >
        <div
          class="mb-32 flex flex-col space-y-12 md:w-1/2"
        >
          <h1
            class="max-w-md text-center text-4xl font-bold md:text-left md:text-5xl"
          >
            Bring everyone together to build better products
          </h1>
          <p
            class="max-w-sm text-center text-darkGrayishBlue md:text-left"
          >
            Manage makes it simple for software teams to plan day-to-day tasks while keeping the larger team goals in view.
          </p>
          <div
            class="flex justify-center md:justify-start"
          >
            <a
              class="rounded-lg p-2 default md baseline rounded-full bg-brightRed p-3 px-6 pt-2 text-white hover:bg-brightRedLight"
              href="#"
            >
              Get Started
            </a>
          </div>
        </div>
        <div
          class="md:w-1/2"
        >
          <img
            alt=""
            class="w-full"
            data-nimg="1"
            decoding="async"
            height="0"
            loading="lazy"
            src="/images/illustration-intro.svg"
            style="color: transparent;"
            width="0"
          />
        </div>
      </div>
    </section>
    <section
      id="features"
    >
      <div
        class="container flex flex-col px-4 mx-auto mt-10 mb-5 space-y-12 md:flex-row md:space-y-0"
      >
        <div
          class="flex flex-col space-y-12 md:w-1/2"
        >
          <h2
            class="max-w-md text-4xl font-bold text-center md:text-left"
          >
            What's different about Manage?
          </h2>
          <p
            class="max-w-sm text-center text-darkGrayishBlue md:text-left"
          >
            Manage provides all the functionality your team needs, without the complexity. Our software is tailor-made for modern digital product teams.
          </p>
        </div>
        <div
          class="flex flex-col space-y-8 md:w-1/2"
        >
          <div
            class="flex flex-col space-y-3 md:flex-row md:space-y-0 md:space-x-6"
          >
            <div
              class="rounded-l-full bg-brightRedSupLight md:bg-transparent"
            >
              <div
                class="flex items-center space-x-2"
              >
                <div
                  class="rounded-full bg-brightRed px-4 py-2 text-white md:py-1"
                >
                  01
                </div>
                <h3
                  class="text-base font-bold md:mb-4 md:hidden"
                >
                  Track company-wide progress
                </h3>
              </div>
            </div>
            <div>
              <h3
                class="mb-4 hidden text-lg font-bold md:block"
              >
                Track company-wide progress
              </h3>
              <p
                class="text-darkGrayishBlue"
              >
                  See how your day-to-day tasks fit into the wider vision. Go from tracking progress at the milestone level all the way done to the smallest of details. Never lose sight of the bigger picture again.
              </p>
            </div>
          </div>
          <div
            class="flex flex-col space-y-3 md:flex-row md:space-y-0 md:space-x-6"
          >
            <div
              class="rounded-l-full bg-brightRedSupLight md:bg-transparent"
            >
              <div
                class="flex items-center space-x-2"
              >
                <div
                  class="rounded-full bg-brightRed px-4 py-2 text-white md:py-1"
                >
                  02
                </div>
                <h3
                  class="text-base font-bold md:mb-4 md:hidden"
                >
                  Advanced built-in reports
                </h3>
              </div>
            </div>
            <div>
              <h3
                class="mb-4 hidden text-lg font-bold md:block"
              >
                Advanced built-in reports
              </h3>
              <p
                class="text-darkGrayishBlue"
              >
                Set internal delivery estimates and track progress toward company goals. Our customisable dashboard helps you build out the reports you need to keep key stakeholders informed.
              </p>
            </div>
          </div>
          <div
            class="flex flex-col space-y-3 md:flex-row md:space-y-0 md:space-x-6"
          >
            <div
              class="rounded-l-full bg-brightRedSupLight md:bg-transparent"
            >
              <div
                class="flex items-center space-x-2"
              >
                <div
                  class="rounded-full bg-brightRed px-4 py-2 text-white md:py-1"
                >
                  03
                </div>
                <h3
                  class="text-base font-bold md:mb-4 md:hidden"
                >
                  Everything you need in one place
                </h3>
              </div>
            </div>
            <div>
              <h3
                class="mb-4 hidden text-lg font-bold md:block"
              >
                Everything you need in one place
              </h3>
              <p
                class="text-darkGrayishBlue"
              >
                Stop jumping from one service to another to communicate, store files, track tasks and share documents. Manage offers an all-in-one team productivity solution.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section
      id="testimonials"
    >
      <div
        class="max-w-6xl px-5 mx-auto mt-32 text-center"
      >
        <h2
          class="text-4xl font-bold text-center"
        >
          What's Different About Manage?
        </h2>
        <div
          class="flex flex-col mt-24 md:flex-row md:space-x-6"
        >
          <div
            class="flex flex-col items-center p-6 space-y-6 rounded-lg bg-veryLightGray md:w-1/3"
          >
             
            <img
              alt=""
              class="w-16 -mt-14"
              data-nimg="1"
              decoding="async"
              height="0"
              loading="lazy"
              sizes="100vh"
              src="/_next/image?url=%2Fimages%2Favatar-anisha.png&w=3840&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar-anisha.png&w=16&q=75 16w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=32&q=75 32w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=48&q=75 48w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=64&q=75 64w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=96&q=75 96w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=128&q=75 128w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=256&q=75 256w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=384&q=75 384w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Favatar-anisha.png&w=3840&q=75 3840w"
              style="color: transparent;"
              width="0"
            />
            <h5
              class="text-lg font-bold"
            >
              Anisha Li
            </h5>
            <p
              class="text-sm text-darkGrayishBlue"
            >
              Manage has supercharged our teamâ€™s workflow. The ability to maintain visibility on larger milestones at all times keeps everyone motivated.
            </p>
          </div>
          <div
            class="flex flex-col items-center p-6 space-y-6 rounded-lg bg-veryLightGray md:w-1/3"
          >
             
            <img
              alt=""
              class="w-16 -mt-14"
              data-nimg="1"
              decoding="async"
              height="0"
              loading="lazy"
              sizes="100vh"
              src="/_next/image?url=%2Fimages%2Favatar-ali.png&w=3840&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar-ali.png&w=16&q=75 16w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=32&q=75 32w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=48&q=75 48w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=64&q=75 64w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=96&q=75 96w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=128&q=75 128w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=256&q=75 256w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=384&q=75 384w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Favatar-ali.png&w=3840&q=75 3840w"
              style="color: transparent;"
              width="0"
            />
            <h5
              class="text-lg font-bold"
            >
              Alia Bravo
            </h5>
            <p
              class="text-sm text-darkGrayishBlue"
            >
              We have been able to cancel so many other subscriptions since using  Manage. There is no more cross-channel confusion and everyone is  much more focused.
            </p>
          </div>
          <div
            class="flex flex-col items-center p-6 space-y-6 rounded-lg bg-veryLightGray md:w-1/3"
          >
             
            <img
              alt=""
              class="w-16 -mt-14"
              data-nimg="1"
              decoding="async"
              height="0"
              loading="lazy"
              sizes="100vh"
              src="/_next/image?url=%2Fimages%2Favatar-richard.png&w=3840&q=75"
              srcset="/_next/image?url=%2Fimages%2Favatar-richard.png&w=16&q=75 16w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=32&q=75 32w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=48&q=75 48w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=64&q=75 64w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=96&q=75 96w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=128&q=75 128w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=256&q=75 256w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=384&q=75 384w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2Favatar-richard.png&w=3840&q=75 3840w"
              style="color: transparent;"
              width="0"
            />
            <h5
              class="text-lg font-bold"
            >
              Richard Watts
            </h5>
            <p
              class="text-sm text-darkGrayishBlue"
            >
              â€œManage has supercharged our teamâ€™s workflow. The ability to maintain visibility on larger milestones at all times keeps everyone motivated.
            </p>
          </div>
        </div>
        <div
          class="my-16"
        >
          <a
            class="p-3 px-6 pt-2 text-white rounded-full baseline bg-brightRed hover:bg-brightRedLight"
            href="#"
          >
            Get Started
          </a>
        </div>
      </div>
    </section>
    <section
      class="bg-brightRed"
      id="cta"
    >
      <div
        class="container flex flex-col items-center justify-between px-6 py-24 mx-auto space-y-12 md:flex-row md:space-y-0 md:py-12"
      >
        <h2
          class="text-5xl font-bold leading-tight text-center text-white md:max-w-xl md:text-left md:text-4xl"
        >
          Simplify how your team works today
        </h2>
        <div>
          <a
            class="p-3 px-6 pt-2 bg-white rounded-full shadow-2xl baseline text-brightRed hover:bg-gray-900"
            href="#"
          >
            Get Started
          </a>
        </div>
      </div>
    </section>
    <section>
      <footer
        class="bg-veryDarkBlue"
      >
        <div
          class="container mx-auto flex flex-col-reverse justify-between space-y-8 px-6 py-10 md:flex-row md:space-y-0"
        >
          <div
            class="flex flex-col-reverse items-center justify-between space-y-12 md:flex-col md:items-start md:space-y-0"
          >
            <div
              class="mx-auto my-6 text-center text-white md:hidden"
            >
              Â© Copyright 2023, All Rights Reserved
            </div>
            <div>
              <img
                alt=""
                class="h-8 w-full"
                data-nimg="1"
                decoding="async"
                height="0"
                loading="lazy"
                src="/images/logo-white.svg"
                style="color: transparent;"
                width="0"
              />
            </div>
            <div
              class="flex justify-center space-x-4"
            >
              <a
                href=""
              >
                <img
                  alt=""
                  class="h-8 w-8"
                  data-nimg="1"
                  decoding="async"
                  height="0"
                  loading="lazy"
                  src="/images/icon-facebook.svg"
                  style="color: transparent;"
                  width="0"
                />
              </a>
              <a
                href=""
              >
                <img
                  alt=""
                  class="h-8 w-8"
                  data-nimg="1"
                  decoding="async"
                  height="0"
                  loading="lazy"
                  src="/images/icon-youtube.svg"
                  style="color: transparent;"
                  width="0"
                />
              </a>
              <a
                href=""
              >
                <img
                  alt=""
                  class="h-8 w-8"
                  data-nimg="1"
                  decoding="async"
                  height="0"
                  loading="lazy"
                  src="/images/icon-twitter.svg"
                  style="color: transparent;"
                  width="0"
                />
              </a>
              <a
                href=""
              >
                <img
                  alt=""
                  class="h-8 w-8"
                  data-nimg="1"
                  decoding="async"
                  height="0"
                  loading="lazy"
                  src="/images/icon-pinterest.svg"
                  style="color: transparent;"
                  width="0"
                />
              </a>
              <a
                href=""
              >
                <img
                  alt=""
                  class="h-8 w-8"
                  data-nimg="1"
                  decoding="async"
                  height="0"
                  loading="lazy"
                  src="/images/icon-instagram.svg"
                  style="color: transparent;"
                  width="0"
                />
              </a>
            </div>
          </div>
          <div
            class="flex justify-around space-x-32"
          >
            <div
              class="flex h-44 flex-col flex-wrap gap-x-12 gap-y-4 text-white"
            >
              <a
                href="/"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Home
                </div>
              </a>
              <a
                href="/pricing"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Pricing
                </div>
              </a>
              <a
                href="/products"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Products
                </div>
              </a>
              <a
                href="/about"
              >
                <div
                  class="hover:text-brightRed"
                >
                  About
                </div>
              </a>
              <a
                href="/careers"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Careers
                </div>
              </a>
              <a
                href="/community"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Community
                </div>
              </a>
              <a
                href="/privacy-policy"
              >
                <div
                  class="hover:text-brightRed"
                >
                  Privacy Policy
                </div>
              </a>
            </div>
          </div>
          <div
            class="flex flex-col justify-between"
          >
            <form>
              <div
                class="flex space-x-3"
              >
                <input
                  class="flex-1 rounded-full px-4 focus:outline-none"
                  placeholder="Updated in your inbox"
                  type="text"
                />
                <button
                  class="rounded-full bg-brightRed px-6 py-2 text-white hover:bg-brightRedLight focus:outline-none"
                >
                  Go
                </button>
              </div>
            </form>
            <div
              class="hidden text-white md:block"
            >
              <h3>
                Â© Copyright 2023. All Rights Reserved
              </h3>
            </div>
          </div>
        </div>
      </footer>
    </section>
  </div>
</DocumentFragment>
`;
